bokehjs/src/lib/models/tools/actions/save_tool.ts

```diff
@@ -36,7 +43,9 @@ export class SaveToolView extends ActionToolView {
 export namespace SaveTool {
   export type Attrs = p.AttrsOf<Props>
 
-  export type Props = ActionTool.Props
+  export type Props = ActionTool.Props & {
+    filename: p.Property<string | null>
+    }
```

Comment:
This indentation is wrong. Linter should have raised an issue for me, but apparently it doesn't work.